independent-publisher-connectors/Google Chart QR Code/apiDefinition.swagger.json

```diff
@@ -0,0 +1,120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Google Chart QR Code Generator",
+    "description": "This Connector is build arount the Google Chart API to generate QR codes to be used in Power Platform applications",
+    "version": "1.0",
+    "contact": {
+      "name": "Iona Varga",
+      "email": "ionavarga@outlook.com"
+    }
+  },
+  "host": "chart.googleapis.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+  {
+      "propertyName": "Website",
+      "propertyValue": "https://chart.googleapis.com/"
```

Comment:
@Ionavoss please provide a valid URL. This is throwing 404 error. 
![image](https://user-images.githubusercontent.com/97551789/208514134-e2a524c9-abf0-45a6-8063-ac4dd3c52b72.png)